utils/noaslr/libnoaslr/src/lib.rs

```diff
@@ -39,7 +44,65 @@ fn libnoaslr() -> Result<()> {
     Ok(())
 }
 
-#[cfg(any(target_os = "linux", target_os = "android"))]
+#[cfg(not(any(target_os = "linux", target_os = "android")))]
```

Comment:
Just use `target_os = "freebsd"` here? Or could this also work on other unix systems?

Reply:
netbsd might work with few adaptations however make no sense for some (e.g. openbsd does not allow to disable aslr whatsoever) so I ll follow your advice.